services/users.js

```diff
@@ -0,0 +1,18 @@
+const usersRepository = require("../repositories/users");
+
+const bcrypt = require("bcryptjs");
+
+const login = async (body) => {
+    const user = await usersRepository.findByEmail(body.email);
+    if (user) {
+        if(bcrypt.compareSync(body.password, user.password))
```

Comment:
Si no se encuentra el usuario o las contraseñas no coinciden hay que tirar un error
Sería un mismo error para ambas validaciones por una cuestión de seguridad, el clásico "user or password is invalid" o algo así